### PR TITLE
Explicitly set SSL_CERT_FILE to cafile in context for urllib

### PR DESCRIPTION
**Pull request by mikeatlas-r7**
Also adds some logging.

### CODE REVIEW

**Comment by mikeatlas-r7** [comment]
@jschipp-r7 here we are...

**Comment by jschipp-r7** [comment]
@mrinehart-r7 Can you review this too?

**Comment by mikeatlas-r7** [comment]
ready for review @jschipp-r7 @mrinehart-r7

File: komand/dispatcher.py [hunk 1]
```diff
@@ -30,5 +30,14 @@ def __init__(self, config={}):
         logging.info('Using dispatcher config: %s', config)
 
     def write(self, msg):
-        r = requests.post(self.url, json=msg)
-        logging.info('POST %s returned %s', self.url, r.content)
+        try:
+            r = requests.post(self.url, 
+                json=msg, 
+                verify=os.environ['SSL_CERT_FILE'])
```
Comment (mrinehart-r7) [inline komand/dispatcher.py]: `os` needs to be imported or else it can't reference it

File: komand/dispatcher.py [hunk 2]
```diff
@@ -30,5 +30,14 @@ def __init__(self, config={}):
...
+            logging.info('POST %s returned %s', self.url, r.content)
+        except Exception as ex:
+            logging.error('ERROR: POST to %s failed. CA bundle path: %s Exception %s',
+                self.url,
+                os.environ['SSL_CERT_FILE'],
```
Comment (mrinehart-r7) [inline komand/dispatcher.py]: Same as above comment

**Comment by mrinehart-r7** [comment]
LGTM

**Comment by mikeatlas-r7** [comment]
thanks for the pep8 changes

**Pull request merged**